tasks/main.yml

```diff
@@ -20,26 +20,38 @@
 - name: Unzip Sonar.
   unarchive:
     src: "{{ workspace }}/{{ sonar_version_directory }}.zip"
-    dest: /usr/local/
+    dest: "{{ sonar_install_dir }}/"
     copy: no
-    creates: /usr/local/sonar/COPYING
+    creates: "{{ sonar_home }}/COPYING"
 
 - name: Move Sonar into place.
   shell: >
-    mv /usr/local/{{ sonar_version_directory }} /usr/local/sonar
-    creates=/usr/local/sonar/COPYING
+    mv {{ sonar_install_dir }}/{{ sonar_version_directory }} {{ sonar_home }}
+    creates={{ sonar_home }}/COPYING
 
 - include: configure.yml
 
 - name: Add sonar symlinks for service management.
   file:
-    src: /usr/local/sonar/bin/linux-x86-64/sonar.sh
+    src: "{{ sonar_home }}/bin/linux-x86-64/sonar.sh"
     dest: "{{ item }}"
     state: link
   with_items:
     - /usr/bin/sonar
     - /etc/init.d/sonar
 
+- name: deploy sonar plugins
+  maven_artifact:
```

Comment:
One note—this module is only available in 2.0 or later, so we also need to up the requirement inside `meta/main.yml` to 2.0 (I'm definitely not opposed to doing that!).